### PR TITLE
Created a project governance doc

### PR DESCRIPTION
**Pull request by Anita-ihuman**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
-[submodule "themes/docsy"]
+ [submodule "themes/docsy"]
```
Comment (chipzoller) [inline .gitmodules:1]: Added space here.

**Comment by chipzoller** [comment]
You punched up the Docsy theme. These updates can sometimes be risky unless you do a full site regression test. Did you go through all the functionality when you updated the submodule?

File: .gitmodules (hunk 1 above)
Comment (Anita-ihuman) [reply .gitmodules:1]: > You punched up the Docsy theme. These updates can sometimes be risky unless you do a full site regression test. Did you go through all the functionality when you updated the submodule?



I am not sure what brought about this error.
Cause I deleted my previous fork and cloned it again when I was making these changes. Only to encounter this.

although the error says there is something wrong with the submodule, I don’t know what exactly is.

Comment (chipzoller) [reply .gitmodules:1]: I'm not sure what you've done, but the refs don't point back to the Docsy repo so builds are failing. Also, you didn't sign off. I think at this point you should just close this PR and open another one taking only the changes you made to the Markdown files with you.

**Pull request closed without merging**